src/kbase/auth/_async/client.py

```diff
@@ -0,0 +1,93 @@
+"""
+A client for the KBase Authentication service.
+"""
+
+import httpx
+import logging
+from typing import Self
+
+from kbase.auth.exceptions import InvalidTokenError, InvalidUserError
+
+# TODO PUBLISH make a pypi kbase org and publish there
+
+
+def _require_string(putative: str, name: str) -> str:
+    if not isinstance(putative, str) or not putative.strip():
+        raise ValueError(f"{name} is required and cannot be a whitespace only string")
+    return putative.strip()
+
+
+def _check_request(r: httpx.Request):
+    try:
+        j = r.json()
```

Comment:
I know that characters are at a premium right now, but could you spare a few more for these variable names?

Reply:
Fixed